### DISCUSSION TITLE
取消自定义组件在编辑器中的点击事件

### DISCUSSION THREAD

**Discussion by hlzzd666** [alibaba/designable#176] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/90434648/147312777-bd091f16-7677-43cb-b538-a052134f34ea.png)
自定义了按钮组件，如果在编辑器中点击按钮会触发按钮的事件，请问怎么才能像其他组件一样不被点击呀

**Comment by janryWang** [comment, marked as answer]
样式控制pointer-events:none
